libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -270,13 +270,7 @@ export class ExecuteBridgeRequest {
   }
 
   private getApiUrl(): string {
-    const apiUrl = process.env.API_ROOT_URL;
-
-    if (!apiUrl) {
-      throw new Error('API_ROOT_URL environment variable is not set');
-    }
-
-    return apiUrl;
+    return `http://localhost:${process.env.PORT}`;
```

Comment:
having this code running in `Worker` ECS instance will the `API` ECS instance will be in the same network?

Reply:
We opted for an approach to only use the localhost loopback for the preview action, where it's guaranteed to be called from the API today.

Reply:
can you elaborate please? not sure i follow how it is not an issue for preview.